scarlet/psf.py

```diff
@@ -10,33 +13,79 @@ def moffat(coords, y0, x0, amplitude, alpha, beta=1.5):
 
         A (1+\frac{(x-x0)^2+(y-y0)^2}{\alpha^2})^{-\beta}
     """
-    Y, X = coords
+    X, Y = np.meshgrid(x, y)
     return (amplitude*(1+((X-x0)**2+(Y-y0)**2)/alpha**2)**-beta)
 
 
-def gaussian(coords, y0, x0, amplitude, sigma):
+def gaussian(y, x, y0=0, x0=0, amplitude=None, sigma=1):
     """Circular Gaussian Function
+
+    Parameters
+    ----------
+    y: array
+        Coordinates to sample the circular gaussian in the y dimension.
+    x: array
+        Coordinates to sample the circular gaussian in the x dimension.
+    amplitude: float
+        Height of the gaussian. If `amplitude` is `None` then
+        `amplitude = 1/(np.pi**2*sigma**2)`
+        (so that the gaussian is normalized).
+    sigma: float
+        Standard deviation of the gaussian.
+
+    Returns
+    -------
+    result: array
+        A 2D circular gaussian sampled at the coordinates `(y_i, x_j)`
+        for all i in `y` and j in `x`.
     """
-    Y, X = coords
+    if amplitude is None:
+        amplitude = 1/(np.pi**2*sigma**2)
+    X, Y = np.meshgrid(x, y)
```

Comment:
It doesn't really matter here, but for consistency `Y, X = np.meshgrid(x,y)`

Reply:
This is actually a flaw in numpy. I believe that the only numpy function that requires `(x, y)` ordering is the `meshgrid` function, since the order matters here (`X` has `len(y)` rows, where each row is `x` and `Y` has `len(x)` columns, where each columns is `y`).

Reply:
I had to check to convince myself of this, but I agree now. That is weird and 🙄